libs/solid/profile/src/lib/components/base/base.component.ts

```diff
@@ -77,6 +77,7 @@ export class BaseComponent implements OnInit, AfterViewInit, OnDestroy {
   public Filter = new UntypedFormControl('');
   public FilterValue = new BehaviorSubject<string>('');
   public SelectedProfile: Profile | null = null;
+  public SelectedProfileShort : ProfileShort = { id : -1, type : undefined};
```

Comment:
Ich glaube, dieses Leerzeichen vor dem `:` verursacht den Formatfehler im Check.

Reply:
Ich glaube, mein Formatting-Tool hat in diesem Commit nicht funktioniert...